# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/ipdl/sync-messages.ini
# Commit: 3f08ccb1f141
# Full Hash: 3f08ccb1f141069bfd90474d6023413303307ec4
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-11-04 09:55:18
# Regressor Bug: 1595994
# File Overlap Count: 1
# Description:
#   Bug 1674043 - P6. Remove sync PDecoderMananger::Supports API. r=mattwoodrow,bryce,mjf,ipc-reviewers,nika
#   
#   In bug 1595994 we attempted to streamline the ability to determine which decoder was available regardless of the process they would be running in. This was subsequently done via the PDMFactory.
#   
#   As there are several JS API that can query which codec are supported, it requires a synchronous mechanism.
# ==============================================================================

diff -r 33563a16a5c3 -r 3f08ccb1f141 ipc/ipdl/sync-messages.ini
--- a/ipc/ipdl/sync-messages.ini	Mon Nov 02 15:32:59 2020 +0000
+++ b/ipc/ipdl/sync-messages.ini	Wed Nov 04 02:22:33 2020 +0000
@@ -884,8 +884,6 @@
 description = legacy sync IPC - please add detailed description
 [PRemoteDecoderManager::PRemoteDecoder]
 description = See Bug 1505976 - investigate changing to async instead of matching GPU pattern
-[PRemoteDecoderManager::Supports]
-description = See Bug 1672072 - We will always need a synchronous call in the worse case scenario, follow-up calls will be cached; however it could be made fully asynchronous under some circumstances..
 [PRemoteDecoderManager::Readback]
 description = legacy sync IPC - please add detailed description
 [PBackground::EnsureRDDProcessAndCreateBridge]
